tests/FSharpPlus.Tests/ComputationExpressions.fs

```diff
@@ -64,7 +64,8 @@ module ComputationExpressions =
                     SideEffects.add "execute this"
             with
             | e -> 
-                SideEffects.add (sprintf "Exception! %s" e.Message)
+                // SideEffects.add (sprintf "Exception! %s" e.Message) // this can be different if run on non-English environment
+                SideEffects.add "Exception! Attempted to divide by zero."
```

Comment:
You could set the culture to be invariant culture when running the code, I think there might be attributes that you can paint the testfixture or tests.

Reply:
```f#
open NUnit.Framework

[<Test;SetCulture("sv-SE")>]
let TestSwedish () =
    Assert.AreEqual("2,9", 2.9.ToString())
[<Test;SetCulture("ja-JP")>]
let TestJapanese () =
    Assert.AreEqual("2.9", 2.9.ToString())
```

Reply:
Though that doesn't seem to change how exceptions are localized, so perhaps better to catch more specific exception `DivideByZeroException` ?